app/parse.py

```diff
@@ -14,5 +24,24 @@ class Course:
     course_type: CourseType
 
 
+def get_single_course(course_soup: BeautifulSoup) -> Course:
+    name = course_soup.select_one(name_class).text
+
+    return Course(
+        name=name,
+        short_description=course_soup.select_one(
+            short_description_class
+        ).text,
+        course_type=CourseType(
+            "part-time" if name[-1] == "й" else "full-time"
```

Comment:
Better to check full name of the course, instead of just 1 letter. It will be more clear